lib/question_component/question/answer.rb

```diff
@@ -0,0 +1,11 @@
+module QuestionComponent
+  class Question
+    class Answer
```

Comment:
Should this be namespaced under `Question`

Reply:
If it's a data structure that is always found as a child object of a question, it should be here.